aws/vault-server-on-eks/configure-vault.sh

```diff
@@ -0,0 +1,51 @@
+#!/usr/bin/env bash
+set -e
+
+# INSTALLING VAULT
+echo "1. Installing vault..."
+curl -sL https://releases.hashicorp.com/vault/1.11.1/vault_1.11.1_linux_amd64.zip -o vault1.zip
+unzip -o vault1.zip
+./vault --version
+
+aws eks --region=us-east-1 update-kubeconfig --name $TF_VAR_cluster_name
+
+VAULT_HOST=$(kubectl get service self-hosted-vault-ui -n self-hosted-vault -o jsonpath='{.status.loadBalancer.ingress[0].hostname}')
+
+echo "VAULT_HOST is $VAULT_HOST"
+
+export VAULT_ADDR=http://$VAULT_HOST:8200
+export VAULT_TOKEN=$TF_VAR_dev_root_token
+
+# Cleaning up anything previously configured
+./vault secrets disable secrets-for-env0/ || true
+./vault auth disable env0-jwt/ || true
+
+# Configuring vault
+./vault secrets enable -path=secrets-for-env0/ kv
+
+# # Bound a ACL policy for permissions
+./vault policy write env0-access - <<EOF
+path "secrets-for-env0/*" {
+    capabilities = ["read", "create", "update"]
+}
+EOF
```

Comment:
`EOF` ?

Reply:
end of file?

Reply:
https://stackoverflow.com/questions/2500436/how-does-cat-eof-work-in-bash

https://en.wikipedia.org/wiki/Here_document